ndsl/dsl/__init__.py

```diff
@@ -6,12 +6,13 @@
 from .dace.dace_config import DaceConfig, DaCeOrchestration
 from .dace.orchestration import orchestrate, orchestrate_function
 from .stencil import (
-    CompilationConfig,
+    CompareToNumpyStencil,
     FrozenStencil,
     GridIndexing,
-    StencilConfig,
     StencilFactory,
+    TimingCollector,
 )
+from .stencil_config import CompilationConfig, RunMode, StencilConfig
 
```

Comment:
Do we not want to expose anything from typing? I guess we use `ndsl.dsl.typing` but I think it makes sense to keep the types on the same level as StencilFactory and other dsl stuff

Reply:
It's a good question. I think we might want to refactor a `dsl.typing` to expose those